model-optimizer/extensions/front/tf/mvn.py

```diff
@@ -88,15 +89,13 @@ def infer(node: Node):
             log.warning('Reduction indices for mean and variance for MVN node {} are not constants'.format(node.name))
```

Comment:
Refactor the `infer` with usage of `node.in_port(x).data.get_value()` and use of Ports and Connections.